src/Html/HtmlCssSelectors.fs

```diff
@@ -62,9 +70,8 @@ module internal HtmlCssSelectors =
                 else
                     inQuotes <- true
                     readString acc t
-            | '\\' :: '\'' :: t when inQuotes ->
```

Comment:
Doesn't the new code supersede it? It would escape the `'` character either in quotes or not in quotes all the same.